realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -327,6 +336,19 @@ public Builder deleteRealmIfMigrationNeeded() {
         }
 
         /**
+         * Setting this will create an in-memory Realm instead of of saving it to disk. In-memory Realms might still use
+         * disk space if memory is running low, but all files created by a in-memory Realm will be deleted when the
+         * Realm is closed.
+         * <p></p>
+         * Note that because in-memory Realms are not persisted, you must be sure to hold on to at least one non-closed
+         * reference to the in-memory Realm object with the specific name as long as you want the data to last.
```

Comment:
Note that because in-memory Realms are not persisted, you must be sure to hold on to at least one non-closed reference to the in-memory Realm object with the specific name. As long as we keep one instance open,  your data is kept.
@nhachicha I feel there are too many "one instance" . What do you think?